homeassistant/components/binary_sensor/ffmpeg.py

```diff
@@ -73,8 +77,16 @@
 
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Create the binary sensor."""
-    from haffmpeg import SensorNoise, SensorMotion
+    from haffmpeg import Test, SensorNoise, SensorMotion
+
+    # check source
+    if config.get(CONF_RUN_TEST):
```

Comment:
If need more time an resource to start (5-8 seconds). So if you know that all is work, you could disable that. But if you mean that should be needed, so I make it not optional.